modules/operation-measure-evaluate-measure/test/blaze/fhir/operation/evaluate_measure_test.clj

```diff
@@ -211,107 +211,270 @@
+                   :url #fhir/uri"url-181501"
+                   :library [#fhir/canonical"library-url-094115"]}]
+            [:put {:fhir/type :fhir/Library :id "0"
+                   :url #fhir/uri"library-url-094115"
+                   :content [library-content]}]]]
+
+          (let [{:keys [status headers body]}
+                @(handler
+                  {:request-method :post
+                   :path-params {:id "0"}
+                   :body
+                   {:fhir/type :fhir/Parameters
+                    :parameter
+                    [{:fhir/type :fhir.Parameters/parameter
+                      :name "periodStart"
+                      :value #fhir/date"2014"}
+                     {:fhir/type :fhir.Parameters/parameter
+                      :name "periodEnd"
+                      :value #fhir/date"2015"}]}})]
+
+            (is (= 201 status))
+
+            (testing "Location header"
+              (is (= "base-url-144638/MeasureReport/AAAAAAAAAAAAAAAA/_history/2"
+                     (get headers "Location"))))
+
+            (given body
+              :fhir/type := :fhir/MeasureReport
+              :status := #fhir/code"complete"
+              :type := #fhir/code"summary"
+              :measure := #fhir/canonical"url-181501"
+              :date := #fhir/dateTime"1970-01-01T00:00:00Z"
+              [:period :start] := #fhir/dateTime"2014"
+              [:period :end] := #fhir/dateTime"2015")))))
+
+    (testing "Async"
       (with-handler [handler]
-        (let [{:keys [status body]}
+        [[[:put
+           {:fhir/type :fhir/Measure :id "0"
+            :url #fhir/uri"url-181501"
+            :library [#fhir/canonical"Library/0"]
+            :scoring (scoring-concept "cohort")
+            :group
+            [{:fhir/type :fhir.Measure/group
+              :population
+              [{:fhir/type :fhir.Measure.group/population
+                :code (population-concept "initial-population")
+                :criteria (cql-expression "InInitialPopulation")}]}]}]
+          [:put
+           {:fhir/type :fhir/Library :id "0"
+            :url #fhir/uri"library-url-094115"
+            :content
+            [#fhir/Attachment
+              {:contentType #fhir/code"text/cql"
+               :data #fhir/base64Binary"bGlicmFyeSBSZXRyaWV2ZQp1c2luZyBGSElSIHZlcnNpb24gJzQuMC4wJwppbmNsdWRlIEZISVJIZWxwZXJzIHZlcnNpb24gJzQuMC4wJwoKY29udGV4dCBQYXRpZW50CgpkZWZpbmUgSW5Jbml0aWFsUG9wdWxhdGlvbjoKICBQYXRpZW50LmdlbmRlciA9ICdtYWxlJwo="}]}]
+          [:put
+           {:fhir/type :fhir/Patient
+            :id "0"
+            :gender #fhir/code"male"}]]]
+
+        (let [{:keys [status headers]}
               @(handler
-                {:params
-                 {"periodStart" "2014"
-                  "periodEnd" "2015"}})]
+                {:request-method :get
+                 :uri "/Measure"
+                 :query-string "measure=url-181501&periodStart=2014&periodEnd=2015"
```

Comment:
Please remove the query string here, because it's just not needed. I know it was there before but that was a mistake by myself.